digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala

```diff
@@ -596,6 +660,16 @@ class VVHClient(vvhRestApiEndPoint: String) {
 
   }
 
+  protected def linkIdFromFeature(attributes: Map[String, Any]): Long = {
+    val linkId = attributes("LINKID").asInstanceOf[BigInt].longValue()
```

Comment:
if you do something like "attributes("LINKID").asInstanceOf[BigInt].longValue()" I think is enough.
You don't need to assign to a variable and then returned, just return the value